frontend/app_flowy/assets/translations/pl-PL.json

```diff
@@ -0,0 +1,145 @@
+{
+  "appName": "AppFlowy",
+  "defaultUsername": "Ja",
+  "welcomeText": "Witaj w @:appName",
+  "githubStarText": "Gwiazdka na GitHub-ie",
+  "subscribeNewsletterText": "Zapisz się do naszego Newslettera",
+  "letsGoButtonText": "Start!",
+  "title": "Tytuł",
+  "signUp": {
+    "buttonText": "Zarejestruj",
+    "title": "Zarejestruj się w @:appName",
+    "getStartedText": "Zaczynamy",
+    "emptyPasswordError": "Hasło nie moze być puste",
+    "repeatPasswordEmptyError": "Powtórzone hasło nie moze być puste",
+    "unmatchedPasswordError": "Hasła nie są takie same",
+    "alreadyHaveAnAccount": "Masz juz konto?",
+    "emailHint": "Email",
+    "passwordHint": "Hasło",
+    "repeatPasswordHint": "Powtórz hasło"
+  },
+  "signIn": {
+    "loginTitle": "Zaloguj do @:appName",
+    "loginButtonText": "Logowanie",
+    "buttonText": "Zaloguj",
+    "forgotPassword": "Zapomniałem hasła?",
+    "emailHint": "Email",
+    "passwordHint": "Password",
+    "dontHaveAnAccount": "Nie masz konta?",
+    "repeatPasswordEmptyError": "Powtórzone hasło nie moze być puste",
+    "unmatchedPasswordError": "Hasła nie są takie same"
+  },
+  "workspace": {
+    "create": "Utwórz przestrzeń",
+    "hint": "przestrzeń robocza",
+    "notFoundError": "Przestrzeni nie znaleziono"
+  },
```

Comment:
Here, you seem to be using "space" instead of "workspace". Are you sure this is ok?

Reply:
Hey Mike,
Yes, I used space on purpose here as the workspace would be "przestrzeń robocza" which might be too long and if it is, there as a hint. But don't worry, it is pretty much interchangeable in Polish. People know, depending on the context, what space will mean. If not, there is always the hint :)

Reply:
Thank you for explaining!